contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -261,7 +262,7 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5, DaoOperator {
     /// @dev claim accumulated fee per platform wallet. Called by any address
     /// @param platformWallet the wallet to claim fee for. Total accumulated fee sent to this wallet.
     /// @return feeWei amount of fee claimed
-    function claimPlatformFee(address platformWallet) external returns (uint256 feeWei) {
+    function claimPlatformFee(address platformWallet) external override nonReentrant returns (uint256 feeWei) {
```

Comment:
notice to limit to 99 chars per line

prettier is off.
but will add it back soon.

Reply:
change as request